lib/core/application/profile/edit_profile_bloc/edit_profile_event.dart

```diff
@@ -0,0 +1,54 @@
+part of 'edit_profile_bloc.dart';
+
+@freezed
+class EditProfileEvent with _$EditProfileEvent {
```

Comment:
I see lots of event with same logic, can't we use keyName, keyValue to update the field ?

Reply:
it's ok for now, but let consider another way when fields are increasing. Agree to approve and enhance later